select.py

```diff
@@ -135,23 +135,26 @@ def init_students():
 ### WORKS WELL ON ITS OWN. EDIT IF YOU KNOW WHAT YOU ARE DOING. 
 def main():
     students = init_students()
+    print("initiated students", students)
     pairs = []
     while not pairs:
-        reset_students(students)
-        pairs = get_pairs(students)
+      reset_students(students)
```

Comment:
Are you sure about this spacing problem? There is similar indentation in other places in the code which you have not fixed. Maybe this issue arises out of the different handling of indentation in different editors.

Reply:
Oh the reason I changed the indentation was because i was using an online IDE and it was indicating me that double tabs have been used for indentation as compared to single. Thus, i changed it. My bad. I'll revert it.

Reply:
It is okay. Leave it as it is.